Dockerfile

```diff
@@ -1,41 +1,11 @@
-FROM python:3.7
+FROM python:3.7-slim-buster
 
-# Install Unix packages
-RUN apt-get update && apt-get install -y \
-    supervisor \
-    cron \
-    nano \
-    git \
-    build-essential \
-    unzip \
-    libaio-dev \
-    && mkdir -p /opt/data/api
+ENV PYTHONDONTWRITEBYTECODE True
+ENV PYTHONUNBUFFERED True
 
-# Oracle Dependencies
-ADD ./vendor /opt/vendor
-
-WORKDIR /opt/vendor
-
-ENV ORACLE_HOME=/opt/oracle/instantclient
-ENV LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$ORACLE_HOME
-
-ENV OCI_HOME=/opt/oracle/instantclient
-ENV OCI_LIB_DIR=/opt/oracle/instantclient
-ENV OCI_INCLUDE_DIR=/opt/oracle/instantclient/sdk/include
-
-# Install Oracle
-RUN mkdir /opt/oracle
-RUN chmod +x /opt/vendor/install.sh
-RUN /opt/vendor/install.sh
-
-RUN mkdir /app
-
-# Copy code
-ADD . /app/
-RUN chmod -R 0644 /app
-WORKDIR /app/
+WORKDIR /app
+ADD . /app
+RUN python -m pip install -r requirements.txt
```

Comment:
ce qu'on fait généralement pour faciliter les itérations, c'est qu'on copie d'abord le requirements.txt pour faire le pip install et ensuite on copie le reste, ça permet d'utiliser la propriété de layers de docker

Reply:
Merci pour l'info !